README.md

```diff
@@ -1,2 +1,12 @@
 # wx-dashboard
 Dashboard for quick and easy access of APRS weather object data.
+
+## Getting Started
+1. Fork and clone repository.
+2. `cd wx-dashboard` and `vagrant up`
```

Comment:
You probably want to add the vagrant-hostmanager plugin dependency info here as well.